Error in Deleting the Project Risks

networkServices.ts:213 
 DELETE http://localhost:3000/api/projectRisks/2 500 (Internal Server Error)

index.tsx:188 Error sending request TypeError: Cannot read properties of undefined (reading 'status')
    at handleDelete (index.tsx:162:20)
﻿

--- a/Servers/utils/projectRisk.utils.ts
+++ b/Servers/utils/projectRisk.utils.ts
@@ -157,6 +157,7 @@ export const deleteProjectRiskByIdQuery = async (
   id: number,
   transaction: Transaction
 ): Promise<Boolean> => {
+  await updateProjectUpdatedByIdQuery(id, "projectrisks", transaction);
   const result = await sequelize.query(
     "DELETE FROM projectrisks WHERE id = :id RETURNING *",
     {
@@ -167,6 +168,5 @@ export const deleteProjectRiskByIdQuery = async (
       transaction
     }
   );
-  await updateProjectUpdatedByIdQuery(id, "projectrisks", transaction);
   return result.length > 0;
 };

